src/Blaster.WebApi/Features/Frontpage/Index.cshtml

```diff
@@ -2,17 +2,16 @@
 {
     <script src="~/frontpage.bundle.js"></script>
 }
-@inject Blaster.WebApi.Security.UserHelper UserHelper
 
 <div id="frontpage-app" v-cloak>
     <banner v-if="showIEBanner" description="Internet Explorer is not a supported browser on the Self Service Portal. Certain functionality is quite likely to not work."></banner>
-</div>
+
 
 <section class="section">
     <div class="container">
         <h1 class="title">Welcome</h1>
         <h2 class="subtitle">
-            Hello @UserHelper.CurrentUser.Name, and welcome to the Cloud Self Service portal.
+            Hello {{getUserName() }}, and welcome to the Cloud Self Service portal.

```

Comment:
Seems to display 'jdog@me.com' when not signed locally. While that is fine locally, it would seem rather out of place if it were to occur in production. Currently this shouldn't be a possible scenario due to the proxies in front of Blaster, making sure that Blaster can't be accessed before being signed in. This however, may not be the case in the near future(assuming everything works out the way we anticipate it).

Reply:
It should display the value set by the UserHelper. So im assuming it will display your personal email in prod. When we remove the auth proxy (and userhelper) it will revert to display either your AD name or guest.